src/Grpc/JsonTranscoding/test/Microsoft.AspNetCore.Grpc.JsonTranscoding.Tests/ServerStreamingServerCallHandlerTests.cs

```diff
@@ -141,6 +146,62 @@ public async Task HandleCallAsync_ErrorWithDetailedErrors_DetailedErrorResponse(
         await callTask.DefaultTimeout();
     }
 
+    [Fact]
+    public async Task HandleCallAsync_HttpBody_WriteMultipleMessages_Returned()
```

Comment:
The tests are a bit over my head, but it looks like there aren't any for the error cases

Reply:
Hmmm, it's difficult to have errors with HttpBody because it just captures the raw body. No serialization happens.